client/src/Account/components/AccountPreviousRewards.tsx

```diff
@@ -177,12 +188,9 @@ const AccountPreviousRewards: FC<AccountPreviousRewardsProps> = ({ rewards }) =>
           return parseFloat(previousRewards.geminiIII.IIIf.earnings).toFixed(2)
         case 'gemini3g':
           return formatUnitsToNumber(
-            rewards
-              .reduce(
-                (acc, reward) => (reward.amount ? acc + BigInt(reward.amount) : acc),
-                BigInt(0),
-              )
-              .toString(),
+            rewards && rewards.amount && rewards.operator
+              ? (BigInt(rewards.amount) - BigInt(rewards.operator)).toString()
+              : '0',
```

Comment:
I think it will be more straight forward if you were to add vote + block rewards, since these are what is being reflected here. Also, on the check I think it is possible that `vote` or `block` is empty, so you should default each to 0 rather than doing the check at the top level.

Reply:
Thank you